mail-client/bower/files/params.patch

```diff
@@ -0,0 +1,4 @@
+--- a/src/Mercury.params	1970-01-01 00:00:00.000000000 +0000
++++ b/src/Mercury.params	2018-11-12 16:21:38.136951570 +0000
+@@ -0,0 +1 @@
++MLLIBS-bower += -ltinfow
```

Comment:
Can't you just `echo` it from the ebuild? Or bother upstream until they make a proper configure check, and patch that in instead?